src/gettingstarted/local/docksal.md

```diff
@@ -0,0 +1,80 @@
+
+```bash
+fin config set --global SECRET_PLATFORMSH_CLI_TOKEN=XXX
+```
+
+## Pulling a Platform.sh project
+
+The Docksal CLI or PHP image ships with the Platform.sh CLI tool. To use the tool and pull a project locally, make sure you have uploaded your SSH key to your Platform.sh account. Once that has been done and a `SECRET_PLATFORMSH_CLI_TOKEN` has been added using the above step. The following command can be ran. Replace `PROJECT_ID` with your project's ID, which can found within the Platform.sh dashboard. Replace `PROJECT_DIRECTORY` with the name of the local directory you'd like the project cloned into. With your SSH key and the `SECRET_PLATFORMSH_CLI_TOKEN` set, you can execute the following command.
+
+If you do not already have Platform.sh CLI installed locally you can use the one that has been installed of the CLI image. The advantage of this would mean that the tool would never have to be installed locally and therefore is one less of a dependency.
+
+```bash
+fin run-cli 'platform get PROJECT_ID -e master PROJECT_DIRECTORY'
+```
+
+And if you do have Platform.sh CLI installed locally you could use that instead.
+
+```bash
+platform get PROJECT_ID -e master PROJECT_DIRECTORY
+```
+
+## Initializing a Platform.sh project
+
+To start a new Docksal project, initialize the configuration with the `fin config generate` command and specify the `docroot` flag.
+
+```bash
+fin config generate --docroot=web
+fin start
+```
+
+## Customizing a Platform.sh project
+
+By default docksal comes configured with a PHP 7.1 Container, a Web Container, and a Database Container. To change these the following variables can be added within your `.docksal/docksal.env` file.
+
+```
+# Apache Versions 2.2 / 2.4
+#WEB_IMAGE='docksal/web:2.1-apache-2.2'
+WEB_IMAGE='docksal/web:2.1-apache-2.4'
+
+# MySQL Version: 5.6 / 5.7 / 8.0
+#DB_IMAGE='docksal/db:1.2-mysql-5.6'
+DB_IMAGE='docksal/db:1.2-mysql-5.7'
+#DB_IMAGE='docksal/db:1.2-mysql-8.0'
+
+# PHP Versions Available 5.6 / 7.0 / 7.1 / 7.2
+#CLI_IMAGE='docksal/cli:2.4-php5.6'
+#CLI_IMAGE='docksal/cli:2.4-php7.0'
+CLI_IMAGE='docksal/cli:2.4-php7.1'
+#CLI_IMAGE='docksal/cli:2.4-php7.2'
+```
+
+If your application has a configuration available from Docksal, you can customize the `.docksal/docksal.yml` file within your project.  This is a docker-compose file and can be customized further as needed for your application, as some customizations are specific to certain applications.
+
+## Downloading MySQL data from Platform.sh into Docksal
+
+In most cases downloading data from Platform.sh and loading it into your project is straightforward. The following commands, run from your application root, will download a compressed database snapshot and load it into the local Docksal database container.
+
+```bash
+fin platform db:dump --gzip -f /tmp/database.sql.gz
+fin exec 'mysql -u user -puser default /tmp/database.sql.gz'
```

Comment:
I think this command is wrong.  It is missing the input redirector and is trying to read a compressed DB straight into MySQL, which it won't like.  I think the following should work:

`fin exec 'gunzip /tmp/database.sql.gz | mysql -u user -puser default'

However, when I run that I get 

```
ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2 "No such file or directory")
```